pyglet/shapes.py

```diff
@@ -116,7 +126,7 @@ class _ShapeGroup(Group):
     sharing the same parent group and blend parameters.
     """
 
-    def __init__(self, blend_src, blend_dest, program, parent=None):
+    def __init__(self, blend_src: int, blend_dest: int, program: ShaderProgram, parent: Optional[Group]=None) -> None:
```

Comment:
why not add a gap between `Optional[Group]` with `=None`
like `parent: Optional[Group] = None`

Reply:
Just makes the line longer and does not add anything